master/buildbot/process/slavebuilder.py

```diff
@@ -188,7 +189,10 @@ def includeSlavename(self, msg):
             msg += " %s" % self.slavename
         return msg
 
-    def ping(self, remote, slavename):
+    def pingTimeout(self):
+        log.msg("ping has timeout")
```

Comment:
Wouldn't it be nicer if we include information about the slave name as well in the log?